src/pages/basic_data/app_admin/OrganizationSearchField.tsx

```diff
@@ -58,6 +59,7 @@ export const OrganizationSearchField = ({
         }
         setSearchTerm('');
       }}
+      defaultValue={currentValue ?? null}
```

Comment:
Litt æsj å bruke `defaultValue` i stedet for `value`, da det fører til en warning om at komponenten går fra uncontrolled til controlled.
Fant ingen god måte å løse det på med `value`, siden dette feltet bare lagrer `id`, mens brukeren skal se navn... så lar det vær slik inntil videre tror jeg 🤔 men burde revisiteres da det er et tegn på dårlig design